tools/introspection/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx_introspection</name>
+    <version>2.0.3</version>
```

Comment:
```suggestion
    <version>2.0.4</version>
```

Reply:
Good catch, thanks!

Reply:
The `VERSION` file still says 2.0.3 ;)

Shouldn't the version be updated by `scripts/iceoryx_release.sh`?

No need to change it now since the script will take care of if later. We might need to define when the version should be set for future PRs.

Reply:
> Shouldn't the version be updated by scripts/iceoryx_release.sh?

@elBoberido @mossmaurice Yes good catch, the version number needs to be changed in all package files, similar to https://github.com/eclipse-iceoryx/iceoryx/pull/1329. This can be done either here or in a separate PR, I'm fine with both.